scripts/__snapshots__/test-react.js.snap

```diff
@@ -2,1903 +2,5811 @@
+    },
+  ],
   "inlinedComponents": 0,
   "optimizedTrees": 1,
 }
 `;
 
 exports[`Test React with JSX input, create-element output Functional component folding Simple with multiple JSX spreads 1`] = `
 ReactStatistics {
+  "evaluatedRootNodes": Array [
+    Object {
+      "children": Array [
+        Object {
+          "children": Array [
+            Object {
+              "children": Array [],
+              "name": "IWantThisToBeInlined",
+              "status": "INLINED",
+            },
+          ],
+          "name": "Button",
+          "status": "INLINED",
+        },
+      ],
+      "name": "App",
+      "status": "ROOT",
+    },
+  ],
   "inlinedComponents": 2,
   "optimizedTrees": 1,
 }
 `;
 
 exports[`Test React with JSX input, create-element output Functional component folding Simple with unary expressions 1`] = `
 ReactStatistics {
+  "evaluatedRootNodes": Array [
+    Object {
+      "children": Array [
+        Object {
+          "children": Array [],
+          "name": "Child",
+          "status": "INLINED",
+        },
+      ],
+      "name": "App",
+      "status": "ROOT",
+    },
+  ],
   "inlinedComponents": 1,
   "optimizedTrees": 1,
 }
 `;
 
 exports[`Test React with JSX input, create-element output fb-www mocks Hacker News app 1`] = `
 ReactStatistics {
+  "evaluatedRootNodes": Array [
+    Object {
+      "children": Array [
+        Object {
+          "children": Array [
+            Object {
+              "children": Array [],
+              "name": "HeaderBar",
+              "status": "INLINED",
+            },
+            Object {
+              "children": Array [],
+              "name": "StoryList",
+              "status": "INLINED",
+            },
+          ],
+          "name": "Unknown",
+          "status": "INLINED",
+        },
+      ],
+      "name": "App",
+      "status": "ROOT",
+    },
+  ],
   "inlinedComponents": 3,
   "optimizedTrees": 1,
 }
 `;
 
 exports[`Test React with JSX input, create-element output fb-www mocks fb-www 1`] = `
 ReactStatistics {
-  "inlinedComponents": 1,
+  "evaluatedRootNodes": Array [
+    Object {
+      "children": Array [
+        Object {
+          "children": Array [],
+          "name": "Unknown",
+          "status": "RENDER_PROPS",
+        },
+      ],
+      "name": "App",
+      "status": "ROOT",
+    },
+  ],
+  "inlinedComponents": 0,
   "optimizedTrees": 1,
 }
 `;
 
 exports[`Test React with JSX input, create-element output fb-www mocks fb-www 2 1`] = `
 ReactStatistics {
+  "evaluatedRootNodes": Array [
+    Object {
+      "children": Array [],
+      "name": "Hello",
+      "status": "ROOT",
+    },
+  ],
   "inlinedComponents": 0,
   "optimizedTrees": 1,
 }
 `;
 
 exports[`Test React with JSX input, create-element output fb-www mocks fb-www 3 1`] = `
 ReactStatistics {
+  "evaluatedRootNodes": Array [],
   "inlinedComponents": 0,
   "optimizedTrees": 0,
 }
 `;
 
 exports[`Test React with JSX input, create-element output fb-www m
```

Comment:
Can we pass this through pretty printer before giving it to jest?

Reply:
I couldn't see how we could, as the snapshot applies its own formatting.